include/envoy/server/configuration.h

```diff
@@ -142,9 +142,9 @@ class Admin {
   virtual const std::string& accessLogPath() PURE;
 
   /**
-   * @return uint32_t the server admin HTTP port.
+   * @return const std::string& the server address.
    */
-  virtual uint32_t port() PURE;
+  virtual const std::string& address() PURE;
```

Comment:
Same thing as the other PR, can we return Address::InstancePtr here (and resolve inside the config) so that we catch resolving errors during config tests.